Sources/SwiftKueryORM/Model.swift

```diff
@@ -919,6 +919,25 @@ public extension Model {
     return orderByArray
   }
 
+  /**
+    This function constructs an array of OrderBy from the Order values
+  */
+  private static func getOrderBy(order: [Order], table: Table) throws -> [OrderBy] {
```

Comment:
This method looks quite familiar to the above method, maybe we could call this method from the above to remove code duplication